harness/tests/experiment/pytorch/test_pytorch_trial.py

```diff
@@ -636,3 +666,77 @@ def test_checkpoint_loading(ckpt: str, istrial: bool):
         assert isinstance(trial, pytorch.PyTorchTrial), type(trial)
     else:
         assert isinstance(trial, torch.nn.Module), type(trial)
+
+
+def make_amp_workloads(
+    trial_class,
+    assert_output_float16=False,
+) -> workload.Stream:
+    trainer = utils.TrainAndValidate()
+    yield from trainer.send(steps=20, validation_freq=1, scheduling_unit=1)
+    training_metrics, _ = trainer.result()
+
+    for metrics, next_metrics in zip(training_metrics[:-1], training_metrics[1:]):
```

Comment:
I think what you are doing here is the right strategy, but still it is tricky enough to warrant a comment explaining why the scale after each step is defined in the next_metrics.

Reply:
Done in 73a6008.